lib/ffi/ogr/srs_api/spatial_reference.rb

```diff
@@ -51,9 +61,16 @@ module SRSAPI
       attach_function :OSRExportToERM,
                       %i[OGRSpatialReferenceH buffer_out buffer_out buffer_out],
                       FFI::OGR::Core::Err
+
+      #------------------
+      # Morphers
+      #------------------
       attach_function :OSRMorphToESRI, %i[OGRSpatialReferenceH], FFI::OGR::Core::Err
       attach_function :OSRMorphFromESRI, %i[OGRSpatialReferenceH], FFI::OGR::Core::Err
 
+      #------------------
+      #
```

Comment:
???

Reply:
i didn't know how to categorize the functions under here, but having a visual separator here was helpful